homework_9.md

```diff
@@ -0,0 +1,72 @@
+## 1. add secondary ip address to you second network interface enp0s8. Each point must be presented with commands and showing that new address was applied to the interface. To repeat adding address for points 2 and 3 address must be deleted (please add deleting address to you homework log) Methods:
+  ### 1. using ip utility (stateless)
+```bash
+[lode00@localhost ~]$ sudo ip addr add 192.168.1.30/24 dev enp0s8 && ip addr show enp0s8
+3: enp0s8: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP group defa                                                                                                   ult qlen 1000
+    link/ether 08:00:27:79:08:6e brd ff:ff:ff:ff:ff:ff
+    inet 10.0.3.15/24 brd 10.0.3.255 scope global noprefixroute dynamic enp0s8
+       valid_lft 77357sec preferred_lft 77357sec
+    inet 192.168.1.30/24 scope global enp0s8
+       valid_lft forever preferred_lft forever
+    inet6 fe80::ccb1:8d61:e989:8d2e/64 scope link noprefixroute
+    valid_lft forever preferred_lft forever
+ ```
+ ```bash
+[lode00@localhost ~]$ sudo ip addr del 192.168.1.30/24 dev enp0s8
+```
+### 2. using centos network configuration file (statefull)
+```bash
+[root@localhost ~]# vim /etc/sysconfig/network-scripts/ifcfg-enp0s8:1
+Type=Ethernet
+BOOTPROTO=none
+ONBOOT=yes
+DEVICE=enp0s8:1
+IPADDR=192.168.1.30
+PREFIX=24
+```
+```bash
+[root@localhost ~]$ reboot
```

Comment:
Is there a more fast and safe way to reload configuration instead of rebooting?

Reply:
[lode00@localhost init.d]$ sudo systemctl restart network   ?

Reply:
Correct
Add this into the report